frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/AppearanceSettings.tsx

```diff
@@ -81,15 +86,19 @@ export const AppearanceSettings = ({
           <StaticEmbedSetupPaneSettingsContentSection
             title={t`Customizing your embed’s appearance`}
           >
-            <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
-              <ExternalLink
-                key="doc"
-                href={docsUrl}
-              >{t`documentation`}</ExternalLink>
-            )} for more.`}</Text>
+            <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${
```

Comment:
I think the copy has already been updated. I couldn't find the exact doc how you're supposed to work with that. But the gist is for the copy that contains links to the documentation to be as isolated as possible, so we could hide the entire sentence. You can see that from the screenshot below that we've split `"Check out the documentation for more." into a separate sentence, so we could hide the entire sentence when we're not supposed to show the doc link as opposed to just remove the link and leave all the copy the same.

![image](https://github.com/metabase/metabase/assets/1937582/44f5ce94-1228-46a3-8ffe-92308fa670d9)